packages/chatbot-server-mongodb-public/src/systemPrompt.ts

```diff
@@ -76,6 +76,10 @@ const fetchPageToolNotes = [
   `If the ${FETCH_PAGE_TOOL_NAME} tool returns the string "${SEARCH_ALL_FALLBACK_TEXT}", you MUST immediately call the ${SEARCH_TOOL_NAME} tool.`,
 ];
 
+const finalAnswerContentNotes = [
+  `If the user provided a URL but you had to use the ${SEARCH_TOOL_NAME} tool to answer their query, you MUST tell the user you didn't find anything on the page provided and that you searched your knowledge base instead.`
```

Comment:
```suggestion
  `If the user provided a URL but you had to use the ${SEARCH_TOOL_NAME} tool to answer their query, you MUST tell the user you didn't find anything on the page provided and that you searched your knowledge base instead.`
```

should you mention that this occurs only when the FETCH_PAGE_TOOL shows no results?